openhtf/__init__.py

```diff
@@ -178,8 +178,9 @@ def Execute(self, test_start=None, loop=None):
     try:
       self._executor.Wait()
     finally:
-      record = self._executor.GetState().GetFinishedRecord()
-      self.OutputTestRecord(record)
+      if self._executor.GetState():
```

Comment:
can you add a comment here that if the framework never transitions from 'INITIALIZING' to 'EXECUTING' then test state never gets set, and there's nothing to output because the test never actually started.